tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/management/ActiveMQServerControlTest.java

```diff
@@ -2715,13 +2715,13 @@ public void testListConnectionsAsJSON() throws Exception {
       assertNotNull(first);
       assertNotNull(second);
 
-      assertTrue(first.getString("connectionID").length() > 0);
-      assertTrue(first.getString("clientAddress").length() > 0);
+      assertTrue(!first.getString("connectionID").isEmpty());
+      assertTrue(!first.getString("clientAddress").isEmpty());
```

Comment:
I think it should actually use assertFalse(foo.isEmpty()) rather than introduce all these assertTrue( !foo.isEmpty()), or else perhaps just leave it the way it was.

Applies here and lots below in this class, along with various other test classes (a couple were updated to flip the assert rather than adding negations)

Reply:
Should all be fixed now.